packages/component-library/src/components/overlay/mt-bare-popover/mt-bare-popover.vue

```diff
@@ -0,0 +1,177 @@
+<template>
+  <slot
+    name="trigger"
+    v-bind="{
+      onClick: toggleDialog,
+      id: `mt-popover__trigger--${id}`,
+      'aria-haspopup': 'dialog' as AriaAttributes['aria-haspopup'],
+    }"
+  />
+
+  <teleport to="body">
+    <div
+      v-if="isVisible"
+      role="dialog"
+      :class="['mt-bare-popover__dialog', [`mt-bare-popover__dialog--width-${width}`]]"
+      ref="dialogRef"
+      tabindex="-1"
+      :style="floatingStyles"
+    >
+      <mt-text v-if="!!title" as="h4" size="s" weight="semibold" class="mt-bare-popover__title"
+        >Title</mt-text
+      >
+
+      <ul class="mt-bare-popover__list">
+        <slot name="default" v-bind="{ closePopover: () => (isVisible = false) }" />
+      </ul>
+    </div>
+  </teleport>
+</template>
+
+<script setup lang="ts">
```

Comment:
Just saw it randomly. Why do you create a new `mt-bare-popover` component instead of using `mt-popover`. The `mt-popover` has the exact same functionality like this component?